proto/minder/v1/minder.proto

```diff
@@ -253,7 +253,7 @@ message PrContents {
         repeated Line patch_lines = 3;
 
         message Line {
-            int32 line_number = 1;
+            int64 line_number = 1;
```

Comment:
I'd leave this as an int32, because I'm terrified of needing more than 2B line numbers for a patch.

Reply:
Yeah, if there's a PR file exceeding 2^32 number of lines, this counter is the last thing to worry about 😄